src/jit/DIRS.proj

```diff
@@ -39,6 +39,7 @@
     <ProjectFile Condition="'$(BuildArchitecture)' == 'amd64'" Include="arm64altjit\arm64altjit.nativeproj" />
     <ProjectFile Condition="'$(BuildArchitecture)' == 'i386'"  Include="protojit\protojit.nativeproj" />
     <ProjectFile Condition="'$(BuildArchitecture)' == 'i386'"  Include="protononjit\protononjit.nativeproj" />
+    <ProjectFile Condition="'$(BuildArchitecture)' == 'i386'"  Include="armelnonjit\armelnonjit.nativeproj" />
```

Comment:
Thanks for adding all the "desktop side" stuff. However, can you leave this line commented out? I can enable it; it will require some additional plumbing that is not in the open.